pkg/localcontroller/manager/dataset.go

```diff
@@ -50,8 +52,11 @@ type DatasetManager struct {
 // Dataset defines config for dataset
 type Dataset struct {
 	*sednav1.Dataset
-	DataSource *DataSource `json:"dataSource"`
-	Done       chan struct{}
+	DataSource     *DataSource `json:"dataSource"`
+	Done           chan struct{}
+	URLPrefix      string
+	Storage        storage.Storage
+	IsLocalStorage bool
```

Comment:
IsLocalStorage may be better to embed into storage.Storage

Reply:
fix it